hadoop-plugin-test/src/main/gradle/negative/missingFields.gradle

```diff
@@ -99,6 +99,10 @@ hadoop {
       depends 'job16'
     }
 
-    targets 'job17'
+    pinotBuildAndPushJob('job20') {
```

Comment:
@jenniferdai Those jobs defined in jobs1.gradle shouldn't interfere with jobs defined in missingFields.gradle, since the jobs are encapsulated in workflows with different names. The Hadoop DSL defines a notion of scope, so that jobs can have the same name, as long as they are within different scopes (such as being in two different workflows with different workflow names).